plugins/woocommerce-admin/webpack.config.js

```diff
@@ -240,6 +242,10 @@ const webpackConfig = {
 						return null;
 					}
 
+					if ( request.startsWith( '@wordpress/dataviews' ) ) {
+						return null;
+					}
```

Comment:
Make sure we bundle the dataviews package in.

Reply:
Is this because the package isn't available in all versions of WP we support? It would be good to have a comment explaining why and when we can remove this.